reachability/mavan/vuln-function-used/README.md

```diff
@@ -0,0 +1,9 @@
+## Example maven project with reachable vulnerabilities
```

Comment:
Maven spelled incorrectly in file path

`reachability/mavan/.....` -> `reachability/maven/.....`